astropy/io/fits/hdu/table.py

```diff
@@ -790,6 +785,30 @@ class BinTableHDU(_TableBaseHDU):
     _extension = 'BINTABLE'
     _ext_comment = 'binary table extension'
 
+    def __init__(self, data=None, header=None, name=None, uint=False):
+        """
+        Parameters
+        ----------
+        header : `Header`
+            header to be used
+        data : array, `FITS_rec` or `astropy.table.Table`
+            data to be used
+        name : str
+            name to be populated in ``EXTNAME`` keyword
+        uint : bool, optional
```

Comment:
Is adding "optional" here necessary? Technically, all keywords are optional but you did not say so for the others.